containers/heavy_forwarder/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal
```

Comment:
Straight copy of https://github.com/openshift/splunk-forwarder-operator/blob/c28e0fb044330d413e18cd7f0a43ab1673d2aa18/containers/heavy_forwarder/Dockerfile